### PR TITLE
Dropped extra columns

### PR DESCRIPTION
**Pull request by arsh1599**
Dropped columns from final dataset having more than 90% empty entries.
Reordered code cells to avoid warning messages

### CODE REVIEW

File: dataset_creation.ipynb [hunk 1]
```diff
@@ -449,82 +3112,101 @@
...
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 40,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pickle\n",
-    "with open('movieDB.pickle', 'wb') as f:\n",
-    "    pickle.dump(movieDB, f)"
+    "#based on columns having more than 85% empty entries\n",
```
Comment (tintindas) [inline dataset_creation.ipynb:453]: new dataframe named as movieDB

File: dataset_creation.ipynb [hunk 2]
```diff
@@ -449,82 +3112,101 @@
...
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 44,
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = pd.DataFrame(movieDB)"
+    "df.to_csv('movieDB_new.csv')"
```
Comment (tintindas) [inline dataset_creation.ipynb]: You're still saving the old dataframe named df to csv here instead of the movieDB dataframe.

**Review by tintindas — CHANGES_REQUESTED** [review]
Could you please do the following?

- [ ] Save movieDB dataframe
- [ ] Clear output before committing

**Comment by arsh1599** [comment]
updated 
1. updates to_csv command
2. removed outputs from python notebook

**Comment by tintindas** [comment]
Thanks!

**Pull request merged**